src/configobj/__init__.py

```diff
@@ -1212,7 +1212,7 @@ def _load(self, infile, configspec):
 
         if isinstance(infile, six.string_types):
             self.filename = infile
-            if os.path.isfile(infile):
+            if os.path.exists(infile):
```

Comment:
Hi @LucidOne,
Probably, it's worth to check whether `infile` is not a directory in order to keep backward compatibility.

```suggestion
            if os.path.exists(infile) and not os.path.isdir(infile):
```